### PR TITLE
Mgirgis stf1639 backported zuul

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    vars:
+      crc_ocp_bundle: 'https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.12.13/crc_libvirt_4.12.13_amd64.crcbundle'
+
+- job:
+    name: stf-crc-ocp_413-local_build-index_deploy
+    parent: stf-crc-local_build-index_deploy
+    description: |
+      Build STF locally and deploy from index on OCP 4.13
+    vars:
+      crc_ocp_bundle: 'https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.13.14/crc_libvirt_4.13.14_amd64.crcbundle'
```
Comment (elfiesmelfie) [inline .zuul.yaml:209]: These should be removed, since they are superceeded by the jobs in L174 and L181.

```suggestion
```

File: .zuul.yaml [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+        - stf-crc-ocp_414-local_build
+        - stf-crc-ocp_412-local_build-index_deploy
+        - stf-crc-ocp_413-local_build-index_deploy
+        - stf-crc-ocp_414-local_build-index_deploy
+
+- project:
+    name: infrawatch/service-telemetry-operator
+    templates:
+      - stf-crc-jobs
+    periodic:
```
Comment (elfiesmelfie) [inline .zuul.yaml:228]: The periodic, nightly_bundle jobs are not necessary on the stable-branch IMHO

File: .zuul.yaml [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+      jobs:
+        - stf-crc-ocp_412-nightly_bundles
+        - stf-crc-ocp_413-nightly_bundles
+        - stf-crc-ocp_414-nightly_bundles
```
Comment (elfiesmelfie) [inline .zuul.yaml:232]:
```suggestion
```

File: build/stf-run-ci/tasks/preflight_checks.yml [hunk 4]
```diff
@@ -19,13 +19,14 @@
...
           oc describe csv $(oc get csv | grep "service-telemetry-operator" | awk '{print $1}') > {{ logfile_dir }}/oc_get_csv_sto.log 2>&1
-          cat {{ logfile_dir }}
+          cat {{ logfile_dir }}/oc_get_csv_sto.log
+      register: csv_sto
 
     - name: "Show service-telemetry-operator CSV information"
       ansible.builtin.debug:
         var: csv_sto.stdout
-
+
```
Comment (elfiesmelfie) [inline build/stf-run-ci/tasks/preflight_checks.yml:28]:
```suggestion

```

**Comment by leifmadsen** [comment]
@elfiesmelfie @mgirgisf is this still required?

**Pull request closed without merging**